lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -567,6 +572,9 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
      * Attempts to set the currently loaded node
      */
     onFolderLoaded(nodePaging: NodePaging): void {
+        setTimeout(() => {
```

Comment:
Is `setTimeout` required here?

Reply:
Unfortunately yes, I added setTimeout because it was giving me following error: 
"expressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'."

I guess it is caused because asynchronous action because onFolderLoaded is called when data loading is finished. I set that timeout without miliseconds because it is enough to fix that error and there is no visible delay thanks that. Other option which I can see is calling change detection manually here so if you prefer that approach I can change it and instead of timeout - call change detection manually. Let me know if you prefer that approach

Reply:
Adding an immediate `setTimeout` is my usual fix for `ExpressionChangedAfterItHasBeenCheckedError` too. You _could_ manually trigger a change detection (I believe it's with `ChangeDetectorRef`), but I generally prefer this solution as it doesn't require an additional injection. `setTimeout` with 0 timeout should just cause the callback to be ran on the next tick, which appears as immediate.

Reply:
Sure guys thanks for explanation :D